CODEOWNERS

```diff
@@ -1,3 +1,9 @@
 * @DataDog/dd-trace-js
 /packages/dd-trace/src/appsec/ @DataDog/appsec-js
 /packages/dd-trace/test/appsec/ @DataDog/appsec-js
+/packages/datadog-plugin-*/ @Datadog/apm-framework-integrations-reviewers-js
+/packages/datadog-instrumentations/ @Datadog/apm-framework-integrations-reviewers-js
```

Comment:
could we exclude CI Visibility specific integrations from this?

Reply:
Here's a list of the ones I see that should be excluded:

Reply:
that's correct